plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/BlacklistScannerTest.java

```diff
@@ -39,7 +39,7 @@
   
   @Before
   public void setUp() throws SAXException {
-    scanner.setDocumentLocator(new Locator2Impl());
+    scanner.setDocumentLocator(new LocatorImpl());
```

Comment:
Locator2 wasn't behaving how I expected it to (was always reporting line/column as 1), so I use a Locator2Impl to get the doc encoding, but a Locator for everything else. If you have any insight into this I'm curious why it wasn't working